<?xml version="1.0"?>
<launch>
  <!-- Color Detector Node (통합: 빨강 + 파랑) -->
  <node name="color_detector" pkg="color_detector" type="color_detector_node" output="screen">
    <!-- 파라미터 설정 -->
    <param name="red_threshold" value="28000"/>
    <param name="blue_threshold" value="28000"/>
    <param name="debug_mode" value="true"/>
  </node>
</launch>
